Source/WebKit/WebProcess/WebPage/Cocoa/TextAnimationController.mm

```diff
@@ -112,17 +118,19 @@
     return std::nullopt;
 }
 
-void TextAnimationController::cleanUpTextAnimationsForSessionID(const WTF::UUID& sessionUUID)
+void TextAnimationController::removeTransparentMarkersForSessionID(const WTF::UUID& sessionUUID)
 {
+    if (m_initialAnimations.contains(sessionUUID))
+        removeTransparentMarkersForTextAnimationID(m_initialAnimations.find(sessionUUID)->value);
+
     auto animationStates = m_activeTextAnimations.take(sessionUUID);
     if (animationStates.isEmpty())
         return;
     for (auto animationState : animationStates)
         removeTransparentMarkersForTextAnimationID(animationState.styleID);
 
-    auto unstyledRange = m_unstyledRanges.find(sessionUUID);
-    if (unstyledRange->value)
-        removeTransparentMarkersForTextAnimationID(unstyledRange->value->styleID);
+    if (m_unstyledRanges.contains(sessionUUID))
+        removeTransparentMarkersForTextAnimationID(m_unstyledRanges.find(sessionUUID)->value->styleID);
```

Comment:
ditto for this; also, what happens if `value` is `nullptr`?

Reply:
That's what the contains check is for, or do I not understand how our HashMaps are implemented?